### DISCUSSION TITLE
How to automatically open screen recording without user confirmation after restarting the system on an Android development board?

### DISCUSSION THREAD

**Discussion by zxm82081372** [rustdesk/rustdesk#9973] [General]
I need to install Rustdesk on the Android self-service machine because it is in kiosk mode, and it is impossible for someone to click to start the service every time I restart. How can I bypass this query?
![image](https://github.com/user-attachments/assets/e553c2f9-38c1-4b9b-9ea4-728825089fbe)

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/1330
